src/components/Tile/Tile.styles.ts

```diff
@@ -0,0 +1,43 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import { memoizeFunction } from 'office-ui-fabric-react/lib/Utilities';
+import { mergeStyleSets, ITheme } from 'office-ui-fabric-react/lib/Styling';
+
+export interface TileClassNameProps {
+  theme: ITheme;
+}
+
+export const getClassNames = memoizeFunction((classNameProps: TileClassNameProps) => {
+  const { theme } = classNameProps;
+
+  return mergeStyleSets({
+    root: {
+      border: `1px solid ${theme.semanticColors.bodyDivider}`,
+      selectors: {
+        '&.y-tile': {
+          position: 'relative',
+          width: '100%',
+          backgroundColor: theme.semanticColors.bodyDivider,
+        },
+        '&.y-tile.y-tile--ratio_21_9': {
+          paddingBottom: '42.85%',
+        },
+        '&.y-tile.y-tile--ratio_16_9': {
+          paddingBottom: '56.25%',
+        },
+        '&.y-tile.y-tile--ratio_4_3': {
+          paddingBottom: '75%',
```

Comment:
If possible, I think it would be better to pass the aspectRatio to getClassNames, and set paddingBottom based on that, rather than adding the `y-tile--ratio...` class.